hack/validate-labels.py

```
@@ -1,45 +1,63 @@
             desc = label.get("description", "")
             if len(desc) > MAX_LENGTH:
-                errors.append(f"repos/{repo}/{label['name']}: {len(desc)} chars - {desc}")
+                errors.append(f"repos/{repo}/{name}: description is {len(desc)} chars (max {MAX_LENGTH})")
+            if name in default_names:
+                errors.append(f"repos/{repo}/{name}: duplicates default label")
+            if name in org_names:
+                errors.append(f"repos/{repo}/{name}: duplicates org-level label from {org}")
 
     if errors:
-        print(f"Labels with descriptions exceeding {MAX_LENGTH} characters:")
+        print(f"Label validation errors:")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)